packages/next/src/server/async-storage/request-async-storage-wrapper.ts

```diff
@@ -119,6 +119,14 @@ export const RequestAsyncStorageWrapper: AsyncStorageWrapper<
 
         return cache.draftMode
       },
+
+      // if you're somehow reading this, please don't use this method.
```

Comment:
Should this be a JSDoc comment so it also shows up on hover when people actually attempt to use it?

Reply:
probably fine, the context itself is not public. I think the name is explicit enough. It's more for those people who read PRs haha

Reply:
would be cool if it was exposed externaly also

Reply:
Can we use a symbol instead of a special key?

Reply:
I thought about it but it seemed even worse, since you could just import the symbol anywhere